### ISSUE TITLE
[FEATURE] Provide an Option to Automatically Hide the Controls

### ISSUE THREAD

**Issue by OlaPessoa** [KrLite/Verticality#18]
### Feature Description

Helloo, it's me once again! could you please add this option in the configs? so that it is toggleable trough the config, and once it's off you can turn it back on trough the config file??

### Propose Reason

Well i just love the mod yet i feel like these arrows at the menu are really irritating

**Comment by KrLite** [comment]
It is practical but... since all you can do is to control the vertical state through the controls, you won't be able to intuitively get them back once they're disabled. When designing these controls, I tried to make them the least disturbing. I'm planning to provide an option to make them visible only when the cursor is on the hotbar, instead of providing an option to totally hide them - that isn't the design.

**Comment by HeaveH0** [comment]
I'm the owner of "OlaPessoa" account, just no longer using it and I'm passing by just to check on you.
I've noticed a 8 months interval of time between the latest updates on all mods up to today, it didn't go unnoticed my friend and I hope you've been doing wonderfully well and progressing a lot in life, eh?

**Comment by KrLite** [comment]
> I'm the owner of "OlaPessoa" account, just no longer using it and I'm passing by just to check on you.
> I've noticed a 8 months interval of time between the latest updates on all mods up to today, it didn't go unnoticed my friend and I hope you've been doing wonderfully well and progressing a lot in life, eh? 

Nope. Just boring school life. You know, in my country we don't have too much spare time for adolescent students.

**Comment by HeaveH0** [comment]
> > I'm the owner of "OlaPessoa" account, just no longer using it and I'm passing by just to check on you.
> > I've noticed a 8 months interval of time between the latest updates on all mods up to today, it didn't go unnoticed my friend and I hope you've been doing wonderfully well and progressing a lot in life, eh?
> 
> Nope. Just boring school life. You know, in my country we don't have too much spare time for adolescent students.

Either way I'm glad you're doing well bud, stay safe and stand firm trough school. God bless you! 🫂
